notebooks/4_cloud_based_data_demo.ipynb

```diff
@@ -26,7 +26,7 @@
   },
```

Comment:
<p>I'd suggest to avoid giving a specific date. Whether we use the cloud-based dataloaders is very much dependent on the usability of the cloud-based service. Maybe something like "we are working on providing the cloud-based dataloader and we'll update this notebook once it's functional."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/sensorium/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>